docs/reference/api.yml

```
@@ -249,20 +247,59 @@ paths:
         all packaging_shapes containing "fe" will be returned. 
         This is useful if you have a search in your application, 
         for a specific product field.
+  '/api/v2/product/{barcode}?fields=images':
+    parameters:
+      - schema:
+          type: string
+        name: barcode
+        in: path
+        required: true
+    get:
+      summary: Get Images for a specific product by barcode (special case of get product)
+      tags:
+        - Read Requests
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  code:
+                    type: string
+                    example: '3017620422003'
+                  product:
+                    type: object
+                    properties:
+                      images:
+                        type: object
+                        # review the regex expression for pattern properties
+                        patternProperties:
+                          '/^\d+$/':
+                            $ref: ./schemas/images/image.yaml
+                          packaging_fr:
```

this also should be a regexp: 

```suggestion
                          '(front|ingredients|nutrition|packaging|other)_\w\w(-\w\w)?:
```